pkgs/tools/text/frogmouth/default.nix

```diff
@@ -15,6 +15,8 @@ python3.pkgs.buildPythonApplication rec {
     hash = "sha256-TMCeIwMEoNDQV9iue4XrdYQMmwofXDvdMMLTZKILQ9Q=";
   };
 
+  patches = [ ./use-xdg-base-dirs.patch ];
```

Comment:
Please add a comment as to why this is needed. Also can thes patch be upstreamed, or can we fetch the patch from upstream?

Reply:
I can fetch the patch from upstream (@figsoda's patch https://github.com/Textualize/frogmouth/pull/59 has rotted; my PR https://github.com/Textualize/frogmouth/pull/85 is still able to be applied.)

It looks like upstream is not disposed to merge it, though, for reasons involving the metadata on `xdg-base-dirs`.

Reply:
I've pushed a new commit that just uses `sed` on the relevant files.